packages/superset-ui-core/src/models/Preset.ts

```diff
@@ -0,0 +1,34 @@
+import Plugin from './Plugin';
+
+export interface PresetConfig {
+  name?: string;
+  description?: string;
+  presets?: Preset[];
+  plugins?: Plugin[];
+}
+
+export class Preset {
```

Comment:
I thought we decided on default exports for classes/components, etc. and named exports for types/interfaces?